run_testing_worker.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Google LLC.
+
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import math
+import os
+import random
+import time
+
+from google.cloud.spanner_v1 import Client
+
+
+def create_test_instance():
+    project = os.getenv(
+        "GOOGLE_CLOUD_PROJECT",
+        os.getenv("PROJECT_ID", "emulator-test-project"),
+    )
+
+    client = Client(project=project)
+
+    config = f"{client.project_name}/instanceConfigs/regional-us-central1"
+
+    name = "spanner-django-test-{}".format(str(int(time.time())))
+
+    instance = client.instance(name, config)
+    created_op = instance.create()
+    created_op.result(120)  # block until completion
+    return name, instance
+
+
+worker_index = int(os.getenv("DJANGO_WORKER_INDEX", 0))
+worker_count = int(os.getenv("DJANGO_WORKER_COUNT", 1))
+
+if worker_index > worker_count:
+    print(
```

Comment:
We might want to replace these `print` lines with `logger.info`, `logger.error` etc. so the script can set the verbosity instead of always printing each line.